gui/src/installer/step/node/electrum.rs

```diff
@@ -84,3 +69,22 @@ impl DefineElectrum {
         Ok(())
     }
 }
+
+pub fn is_electrum_address_valid(value: &str) -> bool {
```

Comment:
I think it would be better to put this function in src/node/electrum.rs if it will be shared by both the app and the installer.